sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 12.2.0 (2020.06)
 
 - Supported quick query. Added a new API `BlockBlobClient.query()`.
+- Supported blob tags.
```

Comment:
Do we have conditional tag support as well? If not do we need to call out that?

Reply:
It's an TODO in following releases.